workbench-core/accounts/src/models/accountSchema.ts

```diff
@@ -0,0 +1,25 @@
+import { z } from 'zod';
+import { HOSTING_ACCOUNT_STATUS } from '../constants/hostingAccountStatus';
+
+// eslint-disable-next-line @rushstack/typedef-var
```

Comment:
rush wants us declare a type for any variable we instantiate instead of having it be calculated by the type system. The Zod types are that get returned are extremely verbose and it would make this code harder to read.